com.synology.SynologyAssistant.yaml

```diff
@@ -0,0 +1,48 @@
+app-id: com.synology.SynologyAssistant
+default-branch: stable
+runtime: org.freedesktop.Sdk
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: synology-assistant
+tags: [proprietary]
+finish-args:
+  - --share=network
+  - --socket=x11
```

Comment:
I believe `--share=ipc` is still needed for anything X11.

Reply:
That wasn't needed for X11 on Wayland, but that sounds correct.